apis/kueue/v1beta1/workload_types.go

```diff
@@ -95,6 +95,13 @@ type PodSet struct {
 	// count is the number of pods for the spec.
 	// +kubebuilder:validation:Minimum=1
 	Count int32 `json:"count"`
+
+	// Total Request holds the pod's resource requirements, the values being
+	// maintained by the Kueue operator.
+	//
+	// The computation can take into account the cluster's LimitRange.
+	// +optional
+	TotalRequests corev1.ResourceList `json:"totalRequests,omitempty"`
```

Comment:
If we move this field into status, we can do the calculations at the time of admission.
This means that we don't need to recreate Workloads. And after admission they will have the information in Status, which the cache can use.